src/detectors/EEMC/CalorimeterHit_factory_EcalEndcapNRecHits.h

```diff
@@ -26,15 +26,15 @@ class CalorimeterHit_factory_EcalEndcapNRecHits : public JFactoryT<edm4eic::Calo
         m_input_tag = "EcalEndcapNRawHits";
 
         // digitization settings, must be consistent with digi class
-        m_capADC=8096;//{this, "capacityADC", 8096};
-        m_dyRangeADC=100. * MeV;//{this, "dynamicRangeADC", 100. * MeV};
-        m_pedMeanADC=400;//{this, "pedestalMean", 400};
-        m_pedSigmaADC=3.2;//{this, "pedestalSigma", 3.2};
+        m_capADC=16384;//{this, "capacityADC", 8096};
+        m_dyRangeADC=20. * GeV;//{this, "dynamicRangeADC", 100. * MeV};
+        m_pedMeanADC=100;//{this, "pedestalMean", 400};
```

Comment:
I did the quick estimation of pedestal in unit of GeV, if I think correctly:
100[mean pedestal in ADC] / 16384[updated ADC dynamic range] * 20GeV = 0.12 GeV
which looks really high.

Reply:
Those are the values in https://github.com/eic/EICrecon/blob/main/src/tools/default_flags_table/reco_flags.py#L182-L209.

Reply:
Those are the values in https://github.com/eic/EICrecon/blob/main/src/tools/default_flags_table/reco_flags.py#L182-L209.